XCUITests/NavigationTest.swift

```diff
@@ -345,6 +345,41 @@ class NavigationTest: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
     }
 
+	func testLongPressOnAddressBar() {
+
+		//Navigate to a URL and perform a long press on the non-highlighted address bar
+		navigator.goto(URLBarOpen)
+		app.textFields["address"].typeText("www.google.com\n")
+		app.textFields["url"].tap()
+		app.textFields["address"].press(forDuration: 2)
```

Comment:
The test is failing for me locally, I had to add here a `waitForExistence(app.menuItems["Select All"], timeout: 3)` as well as `waitUntilPageLoad()`on L#353. Looks like it passed on BB but I think these checks will prevent intermittent failures